### ISSUE TITLE
Pretty printer

### ISSUE THREAD

**Issue by gabrielhdt** [Deducteam/Logipedia#14]
Could the binary of the pretty printer be passed as an argument of logigen?  For instance, one would use `logigen -i json/ -o web --pp /usr/bin/ppt`.  By default, the json object could be printed (although it's not pretty).

**Comment by rprimet** [comment]
Yep the pretty-printer could be an option, especially if we have 2+ choices, but I'd rather have a good default (otherwise the CLI might become tedious to use).

**Comment by gabrielhdt** [comment]
So you want the pretty printer to be in the repository?

**Comment by rprimet** [comment]
No, not at all (it might be a separate package, or it might live in the repository at the start, whichever you prefer) -- I just want to have a default executable name to call it. (logipedia-pp or similar?)

**Comment by rprimet** [comment]
Come to think of it, being able to set the pretty-printer executable through an environment variable (e.g LOGIPEDIA_PP or similar) might also be a good choice (but then we need to define a priority order...)

**Comment by gabrielhdt** [comment]
Ah all right, that's a good idea.  I think we should
1. use an environment variable
2. use the command line flag

so that if both are set, the command line flag is taken into account.  Why not `logipedia-pp` or `logipp`, take the one you prefer.  I have already some prototypes at [logippedia](https://github.com/gabrielhdt/logippedia.git).
